charts/ingress-nginx/ci/daemonset-extra-modules.yaml

```diff
@@ -7,4 +7,5 @@ controller:
     type: ClusterIP
   extraModules:
     - name: opentelemetry
-      image: busybox
+      image:
+        image: busybox
```

Comment:
```suggestion
      image:
        registry: registry.k8s.io
        image: busybox
        tag: latest
```

Reply:
Actually I'm wondering how does can come up as the default `busybox` image from Docker Hub also does not contain the `/usr/local/bin/init_module.sh`.